speech_to_text.py

```diff
@@ -49,7 +57,8 @@ def get_job():
     seconds None will be returned.
     """
     queue = get_todo_queue()
-    jobs = queue.receive_messages(MaxNumberOfMessages=1, WaitTimeSeconds=10)
+    logging.info("looking for messages in the todo queue")
+    jobs = queue.receive_messages(MaxNumberOfMessages=1, WaitTimeSeconds=20)
```

Comment:
20 is the most you can wait when doing "long polling" and is what we are using in speechToTextWF.